urban-os/README.md

```diff
@@ -0,0 +1,52 @@
+# Urban OS Master Chart
```

Comment:
I'll try to update this frequently for correctness, didn't want to slow down a PR for this though.